leakcanary-analyzer/src/test/java/com/squareup/leakcanary/RetainedSizeTest.java

```diff
@@ -21,7 +21,9 @@
   @Parameterized.Parameters public static Collection<Object[]> data() {
     return Arrays.asList(new Object[][] {
         { ASYNC_TASK_PRE_M, 207_407 }, //
+        // TODO why did this change to 145353 with latest perflib?
         { ASYNC_TASK_M, 1_870 }, //
+        // TODO why did this change to 0 with latest perflib?
```

Comment:
Before: Retained: 753 and bitmap: 0
After: Retained: 210978 and bitmap: 0
YourKit says retained size is 944

Reply:
Android Studio does say 210978 as well when loading the heap dump.

Reply:
There's a 984*144 bitmap indirectly held by MainActivity as RippleDrawable.mMaskBuffer. That's an ALPHA_8 bitmap so that makes 141,696 bytes in size (which is the same count we get from Android Studio from that bitmap). Maybe all that extra size comes from some bitmap math based on size? It's weird because the alpha is missing though.

Reply:
The latest perflib has NativeRegistryPostProcessor with:

```
/**
 * In O+, portion of the memory of some object instances (e.g. Bitmap's buffer) has been moved to
 * native. While these are accounted for in the NativeAllocationRegistry in Java space, they do not
 * get blamed on the actualy object instances that are responsible for them. The postprocessor look
 * for these entries in the registry and find their corresponding "owning" instance and add it to
 * their native size.
 */
```